index.js

```diff
@@ -0,0 +1,10 @@
+var express = require( "express" ),
+	app, server, config;
+config = {
+	port: 3000
+};
+app = express();
+app.use ( "/", express.static( "./static" ) );
+server = app.listen( config.port, function() {
+    console.log( "Listening on " + config.port + "..." );
```

Comment:
For the log line, output a valid URL (at least host+port) for Ctrl/Cmd click to work (for command line clients that support it).